frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectServiceDashboard.scenario.ts

```diff
@@ -0,0 +1,214 @@
+      const color = await resiliencyOfMCG.getAttribute('fill');
+      expect(color).toBe(OCP_HEALTH_ICON_COLORS.GREEN);
+    });
+  });
+}
+
+describe('Test Object Data Reduction Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Check for savings value to be in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(savingsValue));
+    const savVal = await savingsValue.getText();
+    const [savDigits] = savVal.split(' ');
+    const numSav = Number(savDigits);
+    if (Number.isNaN(numSav)) {
+      expect(savVal.trim()).toEqual('No Savings');
+    } else {
+      expect(numSav).toBeGreaterThanOrEqual(0);
+    }
+  });
+});
+
+describe('Test Resource Providers card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if resource provider has at least 1 provider', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(resourceProviders));
+    const text = await resourceProviders.getText();
+    expect(text).toBeDefined();
+  });
+});
+
+if (isRGWPresent) {
+  describe('Test RGW Dashboard items are shown', () => {
+    beforeAll(async () => {
+      await loadAndWait();
+      await browser.sleep(10 * SECOND);
+    });
+
+    if (isRGWPresent) {
+      it('Check if dropdown for Service Type is available in Capacity Breakdown Card', async () => {
+        await browser.wait(until.visibilityOf(CapacityBreakdown.serviceTypeDropdown(true)));
+        await click(CapacityBreakdown.serviceTypeDropdown(true));
+        expect(await CapacityBreakdown.serviceTypeItems.ALL.isPresent()).toBeTruthy();
+        expect(await CapacityBreakdown.serviceTypeItems.MCG.isPresent()).toBeTruthy();
+        expect(await CapacityBreakdown.serviceTypeItems.RGW.isPresent()).toBeTruthy();
+      });
+
+      it('Check if MCG Related Breakdown is disabled when RGW/All Service is selected', async () => {
+        await click(CapacityBreakdown.serviceTypeItems.ALL);
+        await click(CapacityBreakdown.serviceTypeDropdown(false));
+        await click(CapacityBreakdown.breakByDropdown(false));
+        expect(await CapacityBreakdown.breakByItems.TOTAL.isEnabled()).toBeTruthy();
+        expect(await CapacityBreakdown.breakByItems.PROJECTS.isEnabled()).toBeFalsy();
+        expect(await CapacityBreakdown.breakByItems.BUCKETS.isEnabled()).toBeFalsy();
+        await click(CapacityBreakdown.breakByDropdown(false));
+        await click(CapacityBreakdown.serviceTypeDropdown(false));
```

Comment:
why do you need to call `breakByDropdown` if `serviceTypeDropdown` is replacing the child as both are using the same selector. - ` $(`.nb-capacity-breakdown-card-header__dropdown.pf-c-select:nth-child(x)` here and at other places too?